templates/media/media-with-message.php

```diff
@@ -0,0 +1,151 @@
+
+	@param object $message Getting parameter response when sending message.
+ **/
+function rtm_add_message_media_params( $message ) {
+	$insert_media_object  = new RTDBModel( 'rtm_media_meta' );
+	$message->media_array = filter_input( INPUT_POST, 'rtm_bpm_uploaded_media' );
+	$media                = explode( ',', $message->media_array );
+	if ( ! empty( $media ) && null !== $media ) {
+		foreach ( $media as $media_id ) {
+			$insert_media_object->insert(
+				[
+					'media_id'   => $media_id,
+					'meta_key'   => 'rtm-bp-message-media', // phpcs:ignore
+					'meta_value' => $message->id, // phpcs:ignore
+				]
+			);
+		}
+	}
+	?>
+	<script>
+		jQuery("#msg-success-bp-msg-media").hide();
+		jQuery(".rtm-media-msg-upload-button").attr("id", "rtm_show_upload_ui");
+		jQuery(".rtm-media-msg-upload-button").html("");
+		jQuery(".rtm-media-msg-upload-button").html("<i class='dashicons dashicons-upload rtmicon'></i>Upload Media File");
+	</script>
+	<?php
+}
+
+/**
+
+	As a result show attached media with message.
+ **/
+function show_rtm_bp_msg_media() {
+	$get_data_object = new RTDBModel( 'rtm_media_meta' );
+	$media_result    = $get_data_object->get( [ 'meta_value' => bp_get_the_thread_message_id() ] );  // phpcs:ignore
+	$url             = explode( 'messages/', sanitize_text_field( wp_unslash( filter_input( INPUT_SERVER, 'REQUEST_URI' ) ) ) );
+	if ( '0' !== $media_result[0]->media_id ) {
+	?>
+	<ul class="rtmedia-list-media rtm-gallery-list clearfix" style = "margin-top: 10px;">
+	<?php
+	foreach ( $media_result as $media_result_array_value ) {
+		$media     = rtmedia_image( 'rt_media_thumbnail', $media_result_array_value->media_id, false );
+		$media_url = $url[0] . '/media/' . $media_result_array_value->media_id . '/';
+		?>
+
+			<li class="rtmedia-list-item" style="display:inline; float: left;" id="<?php echo $media_result_array_value->media_id; // @codingStandardsIgnoreLine?>">
+				<a href="<?php echo esc_attr( $media_url ); ?>" class="<?php echo esc_attr( apply_filters( 'rtmedia_gallery_list_item_a_class', 'rtmedia-list-item-a' ) ); ?>">
+					<div class="rtmedia-item-thumbnail">
+						<img src="<?php echo esc_attr( $media ); ?>" alt="<?php echo esc_attr( apply_filters( 'rtmc_change_alt_text', $alt_text, $rtmedia_media ) ); ?>">
+					</div>
+				</a>
+			</li>
+
+		<?php } ?>
+	</ul>
+	<?php
+	}
+}
+
+// Adding rtMedia uploader in BuddyPress compose message form.
+add_action( 'bp_after_messages_compose_content', 'rtm_bp_message_media_add_upload_media_button' );
+// Adding rtMedia uploader in BuddyPress Send Reply form.
+add_action( 'bp_after_message_reply_box', 'rtm_bp_message_media_add_upload_media_button' );
+// Handling BuddyPress send message action by adding MEDIA ID.
+add_action( 'messages_message_sent', 'rtm_add_message_media_params' );
+// Showing media below BuddyPress message.
+add_action( 'bp_after_message_content', 'show_rtm_bp_msg_media' );
```

Comment:
add new line at the end of file

Reply:
Add new line after each `add_action()` call.